### ISSUE TITLE
Resizing columns from the content, not just from the header

### ISSUE THREAD

**Issue by andreivanea** [olifolkerd/tabulator#189]
Is it possible to resize the columns from within the table body? It would be nice to give the users the ability to pull on a cells (right) border in order to increase the width of the entire column.

**Comment by olifolkerd** [comment]
At present this isn't possible,

Generally speaking most tables only allow for column adjustment by the header as it simplifies controls. Adding cell level column adjustment could interfere with a number of different functions such as callbacks on row clicks, cell clicks, dbl clicks, context clicks, and on interactions with a number of the visual formatters.

I will consider it as a feature for the future, but at this stage i dont think it would be the best fit for tabulator.

Thanks for your suggestion

Cheers

Oli

**Comment by andreivanea** [comment]
Hey, 

Thank you for your answer. I agree with everything you said. I certainly didn't see this as "top level feature". I've implemented this in a grid component I wrote a few years back (also using jQueryUI) and remember that it wasn't a trivial task; the main "focus point" was being sure that the resize event was being triggered before clicks & co., so that propagation could be halted.

Anyway, thank your for taking it into consideration.
